tests/views/index.jade

```diff
@@ -1,12 +1,14 @@
 extends layout
 
 append scripts
+    script(src='../node_modules/babel-polyfill/dist/polyfill.js')

```

Comment:
yeah keep it in. Given I'm an IE/MSEdge MVP I _should_ test in it, but it annoys me that they won't fix the multi-key index bug (well, implement that feature) so I can't add automated tests in Sauce Labs that cover Edge (maybe I should add FF now...)